martin-mbtiles/src/mbtiles_queries.rs

```diff
@@ -1,6 +1,7 @@
-use crate::errors::MbtResult;
 use sqlx::{query, SqliteExecutor};
 
+use crate::errors::MbtResult;
```

Comment:
Is this because the imports of things that are in the crate should be kept separate? It looks like if we use `rustfmt-nightly` we can configure this by default by specifying 
```
reorder_imports_in_group = true
```
in the `rustfmt.toml` file ([link1](https://github.com/rust-lang/rustfmt/issues/298#issuecomment-331481386), [link2](https://docs.rs/crate/rustfmt-nightly/0.3.4/source/Configurations.md)).

Reply:
@upsicleclown do you mean why it has been moved to a separate line separated by a blank line?  We have a few config options already set - https://github.com/maplibre/martin/blob/main/rustfmt.toml - but I don't want to enable it "by default" because the stable rustfmt throws an error if rustfmt has some settings it does not support. So keeping it commented out.

Note that `reorder_imports_in_group` seem to have been removed - try adding it to rustfmt and running `cargo +nightly fmt` - and it will complain.

I added a `fmt2` just target to do it from CLI